superchain/superchain.go

```diff
@@ -418,6 +489,20 @@ func (c ContractVersions) VersionFor(contractName string) (string, error) {
 		version = c.OptimismPortal
 	case "SystemConfig":
 		version = c.SystemConfig
+	case "DisputeGameFactory":
+		version = c.DisputeGameFactory
+	case "DelayedWETH":
+		version = c.DelayedWETH
+	case "AnchorStateRegistry":
+		version = c.AnchorStateRegistry
+	case "FaultDisputeGame":
+		version = c.FaultDisputeGame
+	case "PermissionedDisputeGame":
+		version = c.PermissionedDisputeGame
+	case "MIPS":
+		version = c.MIPS
+	case "PreimageOracle":
+		version = c.PreimageOracle
```

Comment:
nit, everywhere else you keep the same order, and MIPS comes last after PreimageOracle, which is nice to read and makes things easier to verify. Here the last two are swapped.